internal/node/node_repositories.bzl

```diff
@@ -39,6 +39,7 @@ _YARN_VERSIONS = {
     "1.19.1": ("yarn-v1.19.1.tar.gz", "yarn-v1.19.1", "34293da6266f2aae9690d59c2d764056053ff7eebc56b80b8df05010c3da9343"),
     "1.22.4": ("yarn-v1.22.4.tar.gz", "yarn-v1.22.4", "bc5316aa110b2f564a71a3d6e235be55b98714660870c5b6b2d2d3f12587fb58"),
     "1.22.10": ("yarn-v1.22.10.tar.gz", "yarn-v1.22.10", "7e433d4a77e2c79e6a7ae4866782608a8e8bcad3ec6783580577c59538381a6e"),
+    "1.22.11": ("yarn-v1.22.11.tar.gz", "yarn-v1.22.11", "2c320de14a6014f62d29c34fec78fdbb0bc71c9ccba48ed0668de452c1f5fe6c"),
     # When adding a new version. please update /docs/install.md
```

Comment:
What's in docs/install.bzl? Should it be updated?

Reply:
comment is just out of date I think.

docs/install.md says
> The available versions are documented on the `node_repositories` rule in the [Built-ins](Built-ins).